coops-ui/src/mixins/entityFilter-mixin.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Vue } from 'vue-property-decorator'
```

Comment:
Not a big deal, but I think the name of this file should be `entity-filter-mixing.ts`.